### PR TITLE
Restructured camera specific documentation

### LINKED ISSUE #3997: [Support]: Add a Wiki with configuration help
### Describe the problem you are having

I recently installed Frigate and am quite happy with the possibilities and flexibility.

My Setup:
* Unraid/Docker
* Google Coral
* Annke C800 (2x)
* 3-5 Cameras to come

Still work todo like better event handling/notifications via ioBroker, etc. What is quite annoying is to find proper documentation, like camera model specific options, working in a non home assistant environment, FAQ, etc. It helps to search the discussions, but structure is lacking. I suggest enabling the Wiki feature in GitHub to work on documentation (distilled from discussions) together. I'm more than willing to contribute, as I still have some cameras to go.

### Version

0.11.0-3846A13

### Frigate config file

```yaml
Not relevant
```

### Relevant log output

```shell
Not relevant
```

### FFprobe output from your camera

```shell
Not relevant
```

### Frigate stats

```json
Not relevant
```

### Operating system

UNRAID

### Install method

Docker CLI

### Coral version

USB

### Network connection

Wired

### Camera make and model

Annke C800

### Any other information that may be helpful

_No response_

### PR DESCRIPTION
**Pull request by menkej**
* Make room for manufacture specific docs, fixes #3997 
* Added initial (more or less) working setup for Annke C800 camera

### CODE REVIEW

File: docs/docs/configuration/camera_specific.md [hunk 1]
```diff
@@ -56,7 +56,70 @@ ffmpeg:
...
+### UDP Only Cameras
+
+If your cameras do not support TCP connections for RTSP, you can use UDP.
+
+```yaml
+ffmpeg:
+  input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport udp -timeout 5000000 -use_wallclock_as_timestamps 1
+```
+
+### Unifi Protect Cameras
```
Comment (NickM-27) [inline docs/docs/configuration/camera_specific.md:77]: This should go under the model/vensor specific setup section

File: docs/docs/configuration/camera_specific.md [hunk 2]
```diff
@@ -56,7 +56,70 @@ ffmpeg:
...
+```yaml
+ffmpeg:
+  output_args:
+    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -ar 44100 -c:a aac
+    rtmp: -c:v copy -f flv -ar 44100 -c:a aac
+```
+
+### Model/vendor specific setup
+
+#### Annke C800
```
Comment (NickM-27) [inline docs/docs/configuration/camera_specific.md:90]: Is there anything specific about the Annke in this case? Seems like these args would be relevant for all H.265 only cameras.

File: docs/docs/configuration/camera_specific.md [hunk 3]
```diff
@@ -56,7 +56,70 @@ ffmpeg:
...
+        - path: rtsp://user:password@camera-ip:554/H264/ch1/main/av_stream # <----- Update for your camera
+          roles:
+            - detect
+            - record
+            - rtmp
+    rtmp:
+      enabled: False # <-- RTMP should be disabled if your stream is not H264
+    detect:
+      width: 3840 # <---- update for your camera's resolution
+      height: 2160 # <---- update for your camera's resolution
```
Comment (NickM-27) [inline docs/docs/configuration/camera_specific.md:112]:
```suggestion
    detect:
      width: # <---- update for your camera's resolution
      height: # <---- update for your camera's resolution
```

Comment (NickM-27) [reply docs/docs/configuration/camera_specific.md:112]: We never recommend running detect on the high res stream.

File: docs/docs/configuration/camera_specific.md [hunk 4]
```diff
@@ -56,7 +56,70 @@ ffmpeg:
...
+
+    objects:
+      track:
+        - person
```
Comment (NickM-27) [inline docs/docs/configuration/camera_specific.md:116]:
```suggestion
```

Comment (NickM-27) [reply docs/docs/configuration/camera_specific.md:116]: I don't think this is needed as part of the camera specific args.

File: docs/docs/configuration/camera_specific.md [hunk 5]
```diff
@@ -56,7 +56,70 @@ ffmpeg:
...
+### Model/vendor specific setup
+
+#### Annke C800
+This camera is H.265 only. To be able to play clips on a Mac the H.265 stream has to be repackaged and the audio stream has to be converted to aac. Unfortunately direct playback of in the browser is not working (yet), but the downloaded clip can be played locally.
+
+```yaml
+cameras:
+  annkec800: # <------ Name the camera
+    ffmpeg:
+      hwaccel_args: -hwaccel vaapi -hwaccel_device /dev/dri/renderD128 -hwaccel_output_format yuv420p
```
Comment (NickM-27) [inline docs/docs/configuration/camera_specific.md:97]:
```suggestion
      hwaccel_args: [] # <----- Replace with your hwaccel args
```

**Review by NickM-27 — CHANGES_REQUESTED** [review]

**Comment by NickM-27** [comment]
As this is just a docs update, you'll probably want to retarget `master` branch with this PR

**Comment by menkej** [comment]
Why did you change the branch from dev to master @NickM-27 ? dev is default in this repo. How are you using these branches? Just curious - want to get it right next time. ;-)

**Comment by NickM-27** [comment]
@menkej `dev` is for new features / work that will be going into a future version of frigate. As these are just docs changes and they are relevant to the current version of frigate, it should target `master` so it will be uploaded to the docs when the PR is merged

**Review by NickM-27 — APPROVED** [review]

**Comment by NickM-27** [comment]
Looks good though, thank you!

**Pull request merged**